renderer/components/LoadingBolt/LoadingBolt.js

```diff
@@ -48,43 +51,79 @@ const FullPageGradient = styled(Flex)`
   );
   background-size: 400% 400%;
   animation: ${gradientMotion} 10s ease infinite;
-  pointer-events: none;
 `
 
+const FullHeightContent = styled(Flex)`
+  width: 100%;
+  height: 100%;
+`
+
+const LoadingCloseButtonWrapper = styled(Box)`
+  height: 40px;
+  cursor: pointer;
+  opacity: 0.6;
+  &:hover: {
+    opacity: 1;
+  }
+`
+
+const LoadingCloseButton = ({ onClose }) => (
+  <Flex justifyContent="flex-end">
+    <LoadingCloseButtonWrapper onClick={onClose} p={2}>
+      <X height={20} width={20} />
+    </LoadingCloseButtonWrapper>
+  </Flex>
+)
+
+LoadingCloseButton.propTypes = {
+  onClose: PropTypes.func,
+}
+
 class LoadingBolt extends React.PureComponent {
   static propTypes = {
+    hasClose: PropTypes.bool,
     isLoading: PropTypes.bool.isRequired,
     message: PropTypes.object,
+    onClose: PropTypes.func,
```

Comment:
Looks like this prop is not required, however it is directly passed into an onClick without a null check

Reply:
Added null handling in `CloseButton`